File: program/program/convert-feet-to-inches/ConvertFeetToInches.java
import java.util.*;
import java.io.*;
public class Main
{
	public static void main(String[] args) {
	    System.out.print("Please enter the feet value: ");
	    Scanner sc=new Scanner(System.in);
	    int feet=sc.nextInt();
	    int inches = feet*12;
	   System.out.print(feet+" feet is "+inches+ " inches");
		
	}
}

File: program/program/convert-feet-to-inches/convert_feet_to_inches.go
package main

import "fmt"

func feet_to_inches(feet float32) {
	inches := feet * 12
	fmt.Printf("%f inches",inches)
    return
    }

func main() {
	var number float32
	fmt.Printf("Please provide the value in feet: ")
	fmt.Scanln(&number)
	feet_to_inches(number)
}
